server/chat-plugins/wifi.tsx

```diff
@@ -141,21 +145,31 @@ class Giveaway extends Rooms.RoomGame {
 		super.destroy();
 	}
 
-	generateReminder() {}
+	generateReminder(joined?: boolean): string | Chat.VNode;
+	generateReminder() {
+		return '';
+	}
 
 	getStyle() {
-		if (this.game === 'BDSP') return `style="background-color: #aa66a9;color: #fff;padding: 2px 4px;"`;
-		return `class="broadcast-blue"`;
+		if (this.game === 'BDSP') return {style: {backgroundColor: '#aa88a9', color: '#fff', padding: '2px 4px'}};
+		return {class: "broadcast-blue"};
```

Comment:
So what was wrong with the one I suggested, where you always send `broadcast-blue` but sometimes override the colors?

Reply:
Nothing, I just like this approach more because I don't see the point in using a class then changing half of the elements the class has

Reply:
Changing half is better than changing all, and makes the code more readable, too.

Reply:
It's also more semantic. Only changing color is better than hard-changing layout.

Reply:
I don't see how this makes the code more readable but ok